auth/authenticationMiddleware.go

```diff
@@ -0,0 +1,81 @@
+package authentication
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/coreos/go-oidc"
+	"golang.org/x/oauth2"
+)
+
+var oauth2Config oauth2.Config
+var verifier *oidc.IDTokenVerifier
+var ctx context.Context
+
+func init() {
+	//Authentication setup
+	configURL := "http://localhost:8080/auth/realms/ubivius"
+	ctx = context.Background()
+	provider, err := oidc.NewProvider(ctx, configURL)
+	if err != nil {
+		log.Println("Auth panic")
+		panic(err)
+	}
+
+	clientID := "ubivius-client"
+	clientSecret := "7d109d2b-524f-4351-bfda-44ecad030eef"
```

Comment:
I'm guessing these will be in env variables soon

Reply:
yeah, but we could let them be this way for now